pyhafas/profile/base/helper/parse_leg.py

```diff
@@ -82,7 +82,9 @@ def parse_leg(
                                 date) if stopover.get('aTimeR') is not None else None,
                             arrival_platform=stopover.get(
                                 'aPlatfR',
-                                stopover.get('aPlatfS', stopover.get('aPltfR', stopover.get('aPltfS', {})).get('txt'))),
+                                stopover.get('aPlatfS', stopover.get('dPltfR', stopover.get('aPltfS', {})).get('txt'))),
```

Comment:
is there a reason for the change `aPltfR` -> `dPltfR`

Reply:
was accidental. fixed